[bash] Failed to build

```shell
/usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2525: warning: Using 'getservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: /usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2522: warning: Using 'setservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: /usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2547: warning: Using 'endservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
collect2: error: ld returned 1 exit status
make: *** [Makefile:583: bash] Error 1
bash failed to install: `make V=0 -j4` exited with 2
```

Just updated `ncurses`, now I meet another issue now
```shell
':
/usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2525: warning: Using 'getservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: /usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2522: warning: Using 'setservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: /usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2547: warning: Using 'endservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
collect2: error: ld returned 1 exit status
make: *** [Makefile:583: bash] Error 1
bash failed to install: `make V=0 -j4` exited with 2
```
Fixed when building `bash` without --enable-static-link

The error solved when I build it again in the second time, this time I get a new error instead
```shell
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: bashline.o: in function `bash_servicename_completion_function':
/usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2525: warning: Using 'getservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: /usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2522: warning: Using 'setservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: /usr/local/tmp/crew/bash-5.1.tar.gz.dir/bash-5.1/bashline.c:2547: warning: Using 'endservent' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
collect2: error: ld returned 1 exit status
make: *** [Makefile:583: bash] Error 1
bash failed to install: `make V=0 -j4` exited with 2
```

> Alternately try using a `ENV['CFLAGS'] = '-fuse-ld=lld'`

It works! But `lld` has a new error while building
```shell
ld.lld: error: unable to find library -ltinfo
collect2: error: ld returned 1 exit status
make: *** [Makefile:583: bash] Error 1
bash failed to install: `make V=0 -j4` exited with 2
```

--- a/packages/bash.rb
+++ b/packages/bash.rb
@@ -8,13 +8,33 @@ class Bash < Package
   source_url 'https://ftpmirror.gnu.org/bash/bash-5.1.tar.gz'
   source_sha256 'cc012bc860406dcf42f64431bcd3d2fa7560c02915a601aba9cd597a39329baa'
 
+  binary_url ({
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bash-5.1-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bash-5.1-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/bash-5.1-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bash-5.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     aarch64: 'b2af2966a519ae1575ac142ea44f9884dd1bb00648f04c123e605368550568d1',
+      armv7l: 'b2af2966a519ae1575ac142ea44f9884dd1bb00648f04c123e605368550568d1',
+        i686: '0c28792af122162259e8d0069e0eaf7709336b32358fc765f0e11df7faaac4c1',
+      x86_64: 'f313ac9ef7294303e75db4bb4fbc26a12346825498f772c8c672cab71e23a460',
+  })
+
   depends_on 'llvm' => :build # Needed only for lld, which appears to allow linking libiconv where ld does not.
   depends_on 'ncurses' => :build # A static enabled ncurses build w/o "--without-normal" is required to build.
 
+  case ARCH
+  when 'i686'
+  @CONFIGUREFLAGS = '--without-bash-malloc'
+  when 'aarch64','armv7l','x86_64'
+  @CONFIGUREFLAGS = '--with-bash-malloc'
+  end
+
   def self.build
-    ENV['LD'] = 'lld'
-    system "./configure #{CREW_OPTIONS} --with-curses --enable-readline \
-      --with-gnu-malloc --enable-mem-scramble --enable-usg-echo-default \
+    system "env CFLAGS='-pipe -flto=auto' CXXFLAGS='-pipe -flto=auto' LDFLAGS='-flto=auto' \
+      ./configure #{CREW_OPTIONS} #{@CONFIGUREFLAGS} --with-curses --enable-readline \
+      --enable-mem-scramble --enable-usg-echo-default \
       --enable-single-help-strings --enable-select \
       --enable-restricted --enable-progcomp --enable-process-substitution \
       --enable-net-redirections --enable-multibyte --enable-job-control \

